java/Insert_array.java

```diff
@@ -0,0 +1,42 @@
+import java.util.Scanner;
+public class Insertarray
+{
+	public static void main(String[] args)
+	{
+	int n,pos,x;
+	Scanner s=new Scanner(System.in);
+	System.out.println("Enter no of elements you want in array:");
+	n=s.nextInt();
+	int a[]=new int[n+1];
+	System.out.println("Enter alll the elements:");
+	for(int i=0;i<n;i++)
+		{

```

Comment:
Please format the code.

If the IDE you use doesn't support that, it's time to get one that does. ;)
You can also use online tools for formatting code, like: [https://codebeautify.org/javaviewer](https://codebeautify.org/javaviewer)